README.md

```diff
@@ -28,13 +35,17 @@ TODO!
 
 Run the deployment.
 
-    $ ansible-playbook -i <inventory> deploy.yml --tags <app>
+```bash
+$ ansible-playbook -i <inventory> deploy.yml --tags <app>
+```
 
 ## Testing
 
 Tests are run via Travis CI's `.travis.yml`. Unit tests can be manually via docker with:
 
-    $ docker run -t -i -v ${PWD}:/data geerlingguy/docker-ubuntu1604-ansible /data/run_tests.sh
+```bash
+$ docker run -t -i -v ${PWD}:/data geerlingguy/docker-ubuntu1604-ansible /data/run_tests.sh
```

Comment:
Oh, this is just a reformatting of what is already there to be explicit with the _language_. As the roles break out of here, I think this loses value and each role is responsible for its own testing